### ISSUE TITLE
Accuracy metrics

### ISSUE THREAD

**Issue by TharanghiLogendran** [keras-team/keras-io#868]
Your tutorial on Video Classification with a CNN-RNN Architecture was very helpful, thank you. Could you maybe give elaborated instructions on how to retrieve other performance metrics? For example, how do I get a sensitivity and specificity for the different classes?

**Comment by SuryanarayanaY** [comment]
Hi @TharanghiLogendran ,

Could you please refer these classes **[SensitivityAtSpecificity](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/SensitivityAtSpecificity)** and  **[SpecificityAtSensitivity](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/SpecificityAtSensitivity)** where you can find Sensitivity of a particular class(label) at a particular specificity and vice versa.

Could you please check and confirm whether this is useful and let us know if you want more details. Thanks!
